rust/kernel/static_key.rs

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
https://docs.kernel.org/staging/static-keys.html#static-keys indicates that `static_key_{false, true}` are deprecated. Should these be named `static_branch_unlikely`?

Reply:
I'm thinking we can make a macro that takes the condition as a block. Then we can have the assembly block call into that block directly, rather than have a labelled break.

Reply:
I think labelled break is fine. LLVM optimises it very well. If you want assembly block calling them directly then I probably need to add a `fallthrough` block to `asm!`?

Reply:
Ok.